tests/integration/test_redact.py

```
@@ -33,5 +33,5 @@ def test_redact_structured(self):
     def test_redact_with_structured_data(self):
         data = {"phone": "415-867-5309"}
 
-        response = self.redact.redact(data)
-        self.assertEqual(response.code, 400)
+        with self.assertRaises(exceptions.ValidationException):
```

Could you add a little comment about what are you testing and why it should raise an exception or not

---

This is on line 32